pkg/command/cluster/clusterupdate/cmd.go

```diff
@@ -95,6 +99,14 @@ func (cmd *command) run() error {
 		cluster.AuthToken = cmd.authToken
 		ok = true
 	}
+	if cmd.forceTLSDisabled.Changed {
+		cluster.ForceTLSDisabled = &cmd.forceTLSDisabled.Val
+		ok = true
+	}
+	if cmd.forceNonSSLPort.Changed {
+		cluster.ForceNonSslPort = &cmd.forceNonSSLPort.Val
+		ok = true
+	}
```

Comment:
This is the part when we overwrite those flags values only when they were specified.

Reply:
right, changed